adaptors/satusehat.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Satusehat Adaptor
+---
+
+## About Satusehat
+
+[Satusehat](https://satusehat.kemkes.go.id/platform/docs/id/playbook/introduction/)
+is a health data exchange ecosystem that connects information systems or
+applications from all members of the Indonesian digital health ecosystem
+including health facilities, regulators, guarantors, and digital service
+providers.
+
+SATUSEHAT uses HL7 [FHIR](https://www.hl7.org/fhir/) (Fast Healthcare Interoperability Resources) in implementing standard data models and Application Programming Interfaces (APIs).
+
+:::info FHIR
+
+FHIR (Fast Healthcare Interoperability Resources) is a global (international) standard that defines the data format and its elements (called " resources ") and an application programming interface (API ) standard for information exchange
+
+:::
+
+Here is an example of how a POST HTTP request for creating an `Encounter` resource looks like in Satusehat with the FHIR format:
+
+
+```
```

Comment:
There's some very strange formatting here - I'm just going to make a quick fix to that